packages/TorneloScoresheet/src/util/arbiterPin.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Given a pin string, return True if the pin is valid
+ */
+export const pinValid = (inputPin: string): boolean => {
+  const arbiterPin = '1234'; //TODO: change to global variable set by user.
+  return inputPin == arbiterPin ? true : false;
```

Comment:
=== in js!
also no need for the ? true : false.
you can just leave it as `return inputPin === arbiterPin`